playbooks/robusta_playbooks/volume_analysis.py

```diff
@@ -0,0 +1,76 @@
+from robusta.api import action, PersistentVolumeEvent, RobustaPod, ObjectMeta, PodSpec, Volume, PersistentVolumeClaimVolumeSource, Container, VolumeMount, List, BaseBlock, MarkdownBlock,Finding,FindingSource,FindingType
+
+
+@action
```

Comment:
This is a very useful action.

I tried to test it on our prometheus PVC, and had an error of "Multi-Attach error for volume", because the Prometheus PVC has AccessMode RWO (Read Write One)
From what I read, it means it can only be attached to one pod.

A solution for this might be, to run the `ls` command, on the pod that is already using the PVC, using `kubectl exec`

@aantn @alikhanxgrid what do you guys think?

Reply:
Sounds good to me. I had some other suggestions in the [original issue])https://github.com/robusta-dev/robusta/issues/222) (see "caveats" there) but this is a simpler solution.